pypeit/reduce.py

```diff
@@ -609,6 +654,62 @@ def local_skysub_extract(self, global_sky, sobjs,
 
         return None, None, None, None, None
 
+    def load_skyregions(self, skymask_init):
```

Comment:
I am not following why there are two ways to load in these sky regions. I am also not convinced that we want this to be a master file, but we should discuss that.